autokeras/utils/utils.py

```diff
@@ -138,3 +138,23 @@ def add_to_hp(hp, hps, name=None):
     class_name = hp.__class__.__name__
     func = getattr(hps, class_name)
     return func(name=name, **kwargs)
+
+
+def serialize_keras_object(obj):
+    if hasattr(tf.keras.utils, "legacy"):
+        return tf.keras.utils.legacy.serialize_keras_object(obj)
+    else:
+        return tf.keras.utils.serialize_keras_object(obj)
+
+
+def deserialize_keras_object(
+    config, module_objects=None, custom_objects=None, printable_module_name=None
+):
+    if hasattr(tf.keras.utils, "legacy"):
+        return tf.keras.utils.legacy.deserialize_keras_object(
```

Comment:
Please append `# pragma: no cover` to this line to mark it as not covered.

Reply:
Fixed.